tests/statistics_test.sh

```diff
@@ -67,6 +72,22 @@ function statistics_Test
   output=$(statistics --invalid-option)
   msg="Invalid parameter: --invalid-option"
   assertEquals "($LINENO)" "$msg" "$output"
+
+  output=$(statistics --day not_a_day 2> /dev/null)
+  msg="Invalid parameter: not_a_day"
+  assertEquals "($LINENO)" "$msg" "$output"
+
+  output=$(statistics --month not_a_month 2> /dev/null)
```

Comment:
Fwiu here, you're testing an empty value which is nice. However, how about adding another test with an invalid month?

Reply:
not_a_month isn't an empty value. I meant it to be an invalid month, somehow. Like someone was trying to invoke `kw statistics --month not_a_month`.

Reply:
Hmm... correct if I'm wrong, but `not_a_month` and `not_a_year` are just a random string that you're passing to `statistics`.  The test that I suggested in my previous comment was using something like `13` or `-3` in the month parameter.